pkg/iphelpers/iphelpers_test.go

```diff
@@ -571,6 +572,190 @@ var _ = Describe("DecIPAddress operations", func() {
 	})
 })
 
+var _ = Describe("GetIPRange operations", func() {
+	It("creates an IPv4 range properly for 30 bits network address", func() {
+
+		_, ipnet, err := net.ParseCIDR("192.168.21.100/30")
+		ip, _ := AddressRange(ipnet)
+		Expect(err).NotTo(HaveOccurred())
+
+		firstip, lastip, err := GetIPRange(net.ParseIP(ip.String()), *ipnet)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(fmt.Sprint(firstip)).To(Equal("192.168.21.101"))
+		Expect(fmt.Sprint(lastip)).To(Equal("192.168.21.102"))
+
+	})
+	It("creates an IPv4 range properly for 24 bits network address with different range start", func() {
+
+		_, ipnet, err := net.ParseCIDR("192.168.2.200/24")
+		ip := net.ParseIP("192.168.2.23") // range start
+
+		Expect(err).NotTo(HaveOccurred())
+
+		firstip, lastip, err := GetIPRange(net.ParseIP(ip.String()), *ipnet)
+		Expect(err).NotTo(HaveOccurred())
+
+		Expect(fmt.Sprint(firstip)).To(Equal("192.168.2.23"))
+		Expect(fmt.Sprint(lastip)).To(Equal("192.168.2.254"))
+
+	})
+	It("creates an IPv4 range properly for 27 bits network address", func() {
+
+		_, ipnet, err := net.ParseCIDR("192.168.2.200/27")
+		ip, _ := AddressRange(ipnet)
+
+		Expect(err).NotTo(HaveOccurred())
+
+		firstip, lastip, err := GetIPRange(net.ParseIP(ip.String()), *ipnet)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(fmt.Sprint(firstip)).To(Equal("192.168.2.193"))
+		Expect(fmt.Sprint(lastip)).To(Equal("192.168.2.222"))
+
+	})
+	It("creates an IPv4 range properly for 24 bits network address", func() {
+
+		_, ipnet, err := net.ParseCIDR("192.168.2.200/24")
+		ip, _ := AddressRange(ipnet)
+
+		Expect(err).NotTo(HaveOccurred())
+
+		firstip, lastip, err := GetIPRange(net.ParseIP(ip.String()), *ipnet)
+
+		Expect(err).NotTo(HaveOccurred())
+
```

Comment:
nit: please remove this newline

Reply:
same, fixed by a later commit, this is just legacy code being moved around